src/components/Profile/MyPosts/MyPosts.jsx

```diff
@@ -5,17 +5,30 @@ import { I18N } from "../../core/constants";
 
 import s from "./MyPosts.module.css";
 
-const MyPosts = () => {
+const MyPosts = (props) => {
+  const postsData = [
```

Comment:
Consider moving this data next to dialogsData, messagesData etc.

Reply:
done